backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/ContextUtils.kt

```diff
@@ -297,8 +297,8 @@ internal class Llvm(val context: Context, val llvmModule: LLVMModuleRef) {
     }
 
     private fun importMemset(): LLVMValueRef {
-        val parameterTypes = cValuesOf(int8TypePtr, int8Type, int32Type, int32Type, int1Type)
-        val functionType = LLVMFunctionType(LLVMVoidType(), parameterTypes, 5, 0)
+        val parameterTypes = cValuesOf(int8TypePtr, int8Type, int32Type, int1Type)
```

Comment:
memset was likely changed in https://github.com/llvm/llvm-project/commit/1e68724d24ba38de7c7cdb2e1939d78c8b37cc0d
The commit message states that alignment should now be specified with an attribute.
We should ensure that proper alignment is applied for our `memset` usages.

Reply:
Fortunately we pointer align object slots where it's used, so likely not a big problem for us.